test/nyc-integration.js

```diff
@@ -525,6 +527,42 @@ describe('the nyc cli', function () {
         })
       })
 
+      it('works with a different working directory', function (done) {
+        const subdir = path.resolve(fixturesCLI, 'subdir')
+        const args = [bin, 'instrument', '--cwd', subdir, './input-dir', './output-dir']
+
+        const proc = spawn(process.execPath, args, {
+          cwd: fixturesCLI,
+          env: env
+        })
+
+        proc.on('exit', function (code) {
+          code.should.equal(0)
+          const target = path.resolve(subdir, 'output-dir', 'index.js')
+          fs.readFileSync(target, 'utf8')
+            .should.match(/console.log\('Hello, World!'\)/)
```

Comment:
I think we should start using tap's snapshot testing feature here, with new tests being written. I think you'll have to replace `it` with something like
```js
const spawn = require('await-spawn')

test('title', t => { 
  return spawn(...).then(() => {

  });
```

Reply:
I'm planning to do a full overhaul of this file soon... but I'm so ___ing swamped right now.

Reply:
We're ignoring this for now, it can be taken care of after the fact so we can get these merged

Reply:
@JaKXz Cool, I figured I'd write code to what's there.  When the snapshot tests start then I'll write to that.

BTW I had a chat with @coreyfarrell about this PR, and it's going to get broken up into an ES6 PR and a`cwd` PR.  So I wouldn't put too much time into reviewing it right now, thanks.